corpus_explorer/utils.py

```diff
@@ -198,3 +203,69 @@ def generate_topic_scatter_plot(topic_coordinates, topic_proportions):
             marker_size=topic_sizes,
         ),
     )
+
+
+def generate_visualization(topic_coordinates, topic_proportions):
+
+    x_coords = topic_coordinates[:, 0]
+    y_coords = topic_coordinates[:, 1]
+
+    # Scale proportion values to adequate Plotly marker size values
+    scaler = MinMaxScaler(feature_range=(MIN_MARKER_SIZE, MAX_MARKER_SIZE))
+    topic_sizes = scaler.fit_transform(topic_proportions.reshape(-1, 1))
+
+    app = dash.Dash()
+
+    slider = dcc.Slider(
+        id='slider_input',
+        min=1,
+        max=2,
+        step=0.1,
+        value=1,
+    )
+
+    graph = dcc.Graph(id='example')
+    graph_layout = go.Layout(
+        title='Plot',
+        yaxis=dict(range=[-0.5, 0.5]),
```

Comment:
How do we know the ranges for the coordinates (both x and y) will be between -0.5 and 0.5? Am I missing something?

Reply:
No you're right, we don't. This just happens to be the range in which the topic coordinates currently fall.

I think we'll want to do something more dynamic later, such as computing the highest, right-most, left-most and lowest coordinates, and define the range based on these values.